### PR TITLE
docs[README]: Add Ozone parameters section

### PR DESCRIPTION
**Pull request by kristinamancini**
Added a section for Ozone's parameters under the code examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +67,22 @@ data = o3.get_multiple_city_air([ARRAY OF CITY NAMES])
 
 ![Gif of ozone.get_multiple_city_air()](/src/media/ozone_get_multiple_city_air_updated.gif)
 
+### Ozone Parameters
+
+Ozone can fetch the following parameters:
+
+ * aqi: air quality index, a measurement of air quality that tells you how clean or polluted the air is
```
Comment (lahdjirayhan) [inline README.md:74]: I would recommend changing aqi into `aqi` to make it more unambiguous that it's a code argument. Same for other parameters as well.

**Review by Sam-damn — CHANGES_REQUESTED** [review]
@kristinamancini i agree with @lahdjirayhan suggestion, it would be great if you could wrap the parameter's names in code segments so that they could stand out more than their description, and Great work!

File: README.md [hunk 2]
```diff
@@ -67,6 +67,22 @@ data = o3.get_multiple_city_air([ARRAY OF CITY NAMES])
 
 ![Gif of ozone.get_multiple_city_air()](/src/media/ozone_get_multiple_city_air_updated.gif)
 
+### Ozone Parameters
```
[the remark this answers is not in the document]
Comment (kristinamancini) [reply README.md:70]: All good! I agree, that wording sounds better.

**Comment by Sam-damn** [comment]
Thank you @kristinamancini

**Comment by lahdjirayhan** [comment]
All clear and thank you for your contribution! @kristinamancini

**Pull request merged**